weepickle-tests/src/test/scala/com/rallyhealth/weepickle/v1/DefaultsTests.scala

```diff
@@ -111,3 +112,27 @@ object OptionTopDropDefaultTests extends TestSuite {
   }
 }
 
+// Exercises com.rallyhealth.weepickle.v1.LowPriorityImplicits.FromFromInput
+object FromInputStringTests extends TestSuite {
+
+  case class A(d: String)
+  implicit val pickler = WeePickle.macroFromTo[A]
+
+  override val tests: Tests = Tests {
+    test("write default FromInput")(
+      {
+        val fromInput: FromInput = FromScala(A(null))
+        FromScala(fromInput).transform(ToJson.string)
+      } ==> """{"d":null}""")
+    test("write non-default FromInput")(
+      {
+        val fromInput: FromInput = FromScala(A("omg"))
+        FromScala(fromInput).transform(ToJson.string)
+      } ==> """{"d":"omg"}""")
+    test("write null FromInput")(
```

Comment:
line for line this test looks the same as "write default FromInput". `A`'s `d` field does not have a default, so what does default mean in that first test?